public/app/plugins/datasource/cloudwatch/specs/datasource.test.ts

```diff
@@ -890,161 +890,6 @@ describe('CloudWatchDatasource', () => {
       });
     });
   });
-
-  function describeMetricFindQuery(query: any, func: any) {
-    describe('metricFindQuery ' + query, () => {
-      const scenario: any = {};
-      scenario.setup = async (setupCallback: any) => {
-        beforeEach(async () => {
-          await setupCallback();
-          const { ds } = getTestContext({ response: scenario.requestResponse });
-          ds.metricFindQuery(query).then((args: any) => {
-            scenario.result = args;
-          });
-        });
-      };
-
-      func(scenario);
-    });
-  }
-
-  describeMetricFindQuery('regions()', async (scenario: any) => {
```

Comment:
none of these tests actually run on main... 🤦

Reply:
ahh good catch haha. Would be cool to get these to pass instead of ripping them out but also if we haven't had them all this time i guess it seems fine to remove and add back in a working bit later.